framegear/yarn.lock

```diff
@@ -3179,6 +3191,7 @@ __metadata:
   version: 0.0.0-use.local
   resolution: "framegear@workspace:."
   dependencies:
+    "@coinbase/onchainkit": "portal:./.."
```

Comment:
Inteeeeerestinggggggggg

What the heck which magic is this?

Reply:
https://yarnpkg.com/protocol/portal

Reply:
Once we finalize probably want to specify the exact version, but this is great for dev

Reply:
Now you're thinking with portals